banco-filmes/collections/BancoFilmes.postman_collection.json

```diff
@@ -99,44 +116,46 @@
 			"response": []
 		},
 		{
-			"name": "findByTitle",
+			"name": "replace",
 			"request": {
-				"method": "GET",
+				"method": "PUT",
 				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\r\n    \"id\": 1,\r\n    \"director\": \"Christopher Nolan\",\r\n    \"genres\": [\r\n        \"ACTION\",\r\n        \"ADVENTURE\"\r\n    ],\r\n    \"imageUrl\": \"www.bancodefilmes.com.br\",\r\n    \"score\": 9.5,\r\n    \"synopsis\": \"A crazy man dressed as a bat kick bandits asses\",\r\n    \"title\": \"Batman The Dark Knight\",\r\n    \"whereToWatch\": [\r\n        \"HBO_MAX\",\r\n        \"NETFLIX\"\r\n    ],\r\n    \"advisoryRating\": \"QUATORZE\"\r\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
 				"url": {
-					"raw": "localhost:8080/movies/find?title=Batman The Dark Knight",
+					"raw": "localhost:8080/movies",
 					"host": [
 						"localhost"
 					],
 					"port": "8080",
 					"path": [
-						"movies",
-						"find"
-					],
-					"query": [
-						{
-							"key": "title",
-							"value": "Batman The Dark Knight"
-						}
+						"movies"
 					]
 				}
 			},
 			"response": []
 		},
 		{
-			"name": "getTopTen",
+			"name": "delete",
 			"request": {
-				"method": "GET",
+				"method": "DELETE",
 				"header": [],
 				"url": {
-					"raw": "localhost:8080/movies/top",
+					"raw": "localhost:8080/movies/1",
 					"host": [
 						"localhost"
 					],
 					"port": "8080",
 					"path": [
 						"movies",
-						"top"
+						"1"
 					]
 				}
 			},
```

Comment:
This modification was to change the order of declarations? It' looks like you changed the functions but i think you only switch them, right?

Reply:
uhum, I grouped them by request type: get, post, delete and update